src/app/pages/properties/my-wishlist/my-wishlist.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, OnInit } from '@angular/core';
+import { PropertiesWishlistResponse, Property } from 'src/app/models/Property';
+import { NgxSpinnerService } from 'ngx-spinner';
+import { ToastrService } from 'ngx-toastr';
+import { PropertiesWishlistService } from 'src/app/services/properties/properties-wishlist/properties-wishlist.service';
+import { Router } from '@angular/router';
+import { BuyPropertyService } from 'src/app/services/properties/properties-wishlist/buy-property/buy-property.service';
+
+
+@Component({
+  selector: 'app-my-wishlist',
+  templateUrl: './my-wishlist.component.html',
+  styleUrls: ['../../profile/transaction-information/sass/transaction-information.component.scss']
+})
+export class MyWishlistComponent implements OnInit {
+  public properties: [];
+  constructor(
+    private toastrService: ToastrService,
+    private spinner: NgxSpinnerService,
+    private propertiesWishlistService: PropertiesWishlistService,
+    private buyPropertyService: BuyPropertyService,
+    private router: Router, ) { }
+
+  ngOnInit() {
+    this.spinner.show();
+    this.propertiesWishlistService.getMyWishlist().subscribe((response: PropertiesWishlistResponse []) => {
+      this.properties = response['data'].property;
```

Comment:
With this kind of implementation, TS throws the following error
> object access via string literals is disallowed

you can use a variable instead of a string literal
```suggestion
      const data = 'sampleData';
      this.properties = response[data].property;
```